privacyidea/api/lib/prepolicy.py

```diff
@@ -1229,9 +1229,9 @@ def is_remote_user_allowed(req, write_to_audit_log=True):
     :param req: The flask request, containing the remote user and the client IP
     :param write_to_audit_log: whether the policy name should be added to the audit log entry
     :type write_to_audit_log: bool
-    :return: a bool value
+    :return: Return a value or REMOTE_USER, can be "disable", "active" or "force".
+    :rtype: basestring
```

Comment:
... for REMOTE_USER...
and please use `str` as type, `basestring` is so Python 2.7...

Reply:
Done in e8b69ceb